tests/Microsoft.Identity.Test.Unit/RequestsTests/SilentRequestTests.cs

```diff
@@ -95,35 +96,11 @@ public void ExpiredTokenRefreshFlowTest()
         }
 
         [TestMethod]
-        public async Task BrokerSilentRequestLocalCacheTestAsync()
-        {
-            //Broker is configured by user and is installed.
-            //Should be pulling from local cache
-            await BrokerSilentRequestTestExecutorAsync(true, true).ConfigureAwait(false);
-        }
-
-        [TestMethod]
-        public async Task BrokerSilentRequestBrokerRequiredTestAsync()
-        {
-            //Broker is not configured by user but is installed
-            await BrokerSilentRequestTestExecutorAsync(false, true).ConfigureAwait(false);
-        }
-
-        [TestMethod]
-        public async Task BrokerSilentRequestBrokerConfiguredButNotInstalledTestAsync()
-        {
-            //Broker is configured by user but is not installed
-            await BrokerSilentRequestTestExecutorAsync(false, true).ConfigureAwait(false);
-        }
-
-        [TestMethod]
-        public async Task BrokerSilentRequestBrokerNotConfiguredAndNotInstalledTestAsync()
-        {
-            //Broker is not configured by user and is not installed
-            await BrokerSilentRequestTestExecutorAsync(false, false).ConfigureAwait(false);
-        }
-
-        public async Task BrokerSilentRequestTestExecutorAsync(bool brokerConfiguredByUser, bool brokerIsInstalledAndInvokable)
+        [DataRow(true, true)] //Broker is configured by user and is installed.
```

Comment:
Can we have a unit test where we mock the broker so that it matches how iOS broker behaves? Have a look at [BrokerTests.cs ](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs#L444)